### DISCUSSION TITLE
Change template wazuh

### DISCUSSION THREAD

**Discussion by aiceware99** [wazuh/wazuh#14196] [General]
How to change the wazuh-dashboard design template and change the name of the wazuh title?

**Comment by mdiego92** [comment]
Hello aiceware99,

Could you please specify what you want to change in the design template?

Regarding the title and logo customization (if needed) please follow these steps:

Edit the `/etc/wazuh-dashboard/opensearch_dashboards.yml` file to include the following lines:

```
opensearch_security.basicauth.login.brandimage: "/plugins/wazuh/assets/logo.svg"
opensearch_security.basicauth.login.title: "Custom title"
opensearch_security.basicauth.login.subtitle: "Custom subtitile"
```

Where the `/plugins/wazuh/assets/logo.svg` path translates to the filesystem's `/usr/share/wazuh-dashboard/plugins/wazuh/public/assets/logo.svg` and this file must be owned by the **wazuh-dashboard** user.

The Wazuh dashboard default logos are in the `/usr/share/wazuh-dashboard/src/core/server/core_app/assets/` folder.

After making the changes they will take effect after restarting the wazuh-dashboard service:

`systemctl restart wazuh-dashboard`

For the images within the plugin you may modify them by specifying a path from within the application itself by going into **Wazuh > Settings > Configuration**, and then scrolling to the bottom of the page.
The paths are relative to `/usr/share/wazuh-dashboard/plugins/wazuh/public/assets/ `.

Please let us know if you have any remaining questions or doubts.

Thank you very much.

**Comment by zeroneaov** [comment]
Hi, can we change the window title of wazuh dashboard as for default it is set to wazuh ? Thank you

**Reply by aliicybersec** [reply]
Have you found a solution to this problem?
